python/ml_croissant/ml_croissant/_src/datasets.py

```diff
@@ -65,6 +65,8 @@ def __post_init__(self):
 
     def records(self, record_set: str) -> Records:
         """Accesses all records belonging to the RecordSet named `record_set`."""
```

Comment:
Maybe add "if it exists" to the function comment?

Reply:
Done.